wellcomeml/ml/bilstm.py

```diff
@@ -133,6 +135,10 @@ def fit(self, X, Y, embedding_matrix=None, *_):
             CALLBACK_DICT[c] if c in CALLBACK_DICT else c
             for c in self.callbacks
         ]
+        if self.early_stopping:
+            early_stopping = tf.keras.callbacks.EarlyStopping(
```

Comment:
That is such a useful callback - I did not know that. Does it just monitor the training loss, or anyway you can make it monitor validation too?

Reply:
It monitors validation loss by default but you can change to any of the available metrics https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/EarlyStopping